### PR TITLE
Notice board: House rule textbox + contact information cards

### PR DESCRIPTION
**Pull request by mikejensen0**
Implements #21 & #22 

Implemented an editable textbox (should only be editable by group creator (waiting for GunDB)) and contact information cards (+ 2 examples). Furthermore, the page is made scrollable (testable by adding more placeholder users)

### CODE REVIEW

File: app/(tabs)/notice.tsx [hunk 1]
```diff
@@ -1,31 +1,72 @@
...
+    </SafeAreaView>
   );
 }
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
+    backgroundColor: 'white',
```
Comment (UnluckyBird) [inline app/(tabs)/notice.tsx:41]: This makes color also white when dark mode is on, maybe either remove this line entirely or make black when dark mode is on if you want to keep the full white background color

**Review by UnluckyBird — APPROVED** [review]

**Comment by UnluckyBird** [comment]
Small suggestion otherwise lgtm :)

**Review by andreasalstrup — APPROVED** [review]
LGTM

File: app/(tabs)/notice.tsx (hunk 1 above)
Comment (andreasalstrup) [reply app/(tabs)/notice.tsx:41]: We ditch dark mode.

**Pull request merged**